plugins/woocommerce/includes/rest-api/Controllers/Version4/Fulfillments/class-wc-rest-fulfillments-v4-controller.php

```
@@ -0,0 +1,767 @@
+				'type'        => 'integer',
+				'required'    => true,
+				'context'     => array( 'view', 'edit' ),
+			),
+			'tracking_number' => array(
+				'description' => __( 'The tracking number to look up.', 'woocommerce' ),
+				'type'        => 'string',
+				'required'    => true,
+				'context'     => array( 'view', 'edit' ),
+			),
+		);
+	}
```

PHPMD (2.15.0)

Avoid unused private methods such as 'get_args_for_get_tracking_number_details'. (Unused Code Rules)

(UnusedPrivateMethod)